tests/cpp_tests/testutils.h

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright (c) 2021 Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022 Microsoft Corporation. All rights reserved.
```

Since this is an entirely new file, this and other new files in this PR should probably be 2022. We have not been very consistent about that in the project, but I just happened to notice this one.